docs/design/architecture_3.0/README.md

```diff
@@ -0,0 +1,145 @@
+![resource manager](./images/resourceManager.png)
+
+## Roadmap
+
+- Stage 1: provide basic features (current delivered)
+- Stage 2: support common features
+- Stage 3: support full features
+
+| **Class**                  | **Sub-Class**       | **Development Stage** |
+| -------------------------- | ------------------- | --------------------- |
+| Service                    | task service        | Stage 1               |
+|                            | extend service      | Stage 3               |
+|                            | image service       | Stage 3               |
+| Runtime handler            | `Virt-Container`    | Stage 1               |
+|                            | `Wasm-Container`    | Stage 3               |
+|                            | `Linux-Container`   | Stage 3               |
+| Endpoint                   | VETH Endpoint       | Stage 1               |
+|                            | Physical Endpoint   | Stage 2               |
+|                            | Tap Endpoint        | Stage 2               |
+|                            | `Tuntap` Endpoint   | Stage 2               |
+|                            | `IPVlan` Endpoint   | Stage 3               |
+|                            | `MacVlan` Endpoint  | Stage 3               |
+|                            | MACVTAP Endpoint    | Stage 3               |
+|                            | `VhostUserEndpoint` | Stage 3               |
+| Network Interworking Model | Tc filter           | Stage 1               |
+|                            | Route               | Stage 1               |
+|                            | `MacVtap`           | Stage 3               |
+| Storage                    | Virtio-fs           | Stage 1               |
+|                            | `nydus`             | Stage 2               |
+| Hypervisor                 | `Dragonball`        | Stage 1               |
+|                            | QEMU                | Stage 2               |
+|                            | ACRN                | Stage 3               |
+|                            | Cloud Hypervisor    | Stage 3               |
+|                            | Firecracker         | Stage 3               |
+
+## FAQ
+
+- Are the "service", "message dispatcher" and "runtime handler" all part of the single Kata 3.x runtime binary?
+
+  <font size=2>Yes. They are components in Kata 3.x runtime. </font><br>
```

Comment:
Please can you remove the HTML tags. afaics they could all be removed and just plain markdown used instead.

Reply:
fixed